db/seeds/abilities.yml

```diff
@@ -1,52 +1,52 @@
 - name: Participate Generally
   internal_id: unrestricted
-  icon: fas fa-heartbeat
-  description: >
+  icon: heartbeat
+  summary: >
     New user restrictions are removed: No limit on posts, vote and comment anywhere
   post_score_threshold: 0.8  # min. +3/-0
   edit_score_threshold: null
   flag_score_threshold: null
 
 - name: Edit Posts
   internal_id: edit_posts
-  icon: fas fa-pen
-  description: >
+  icon: pen
+  summary: >
     Edit all posts (not only own) and review suggested edits to posts
   post_score_threshold: null
   edit_score_threshold: 0.95   # min. +18/-0
   flag_score_threshold: null
 
 - name: Edit Tags
   internal_id: edit_tags
-  icon: fas fa-tags
-  description: >
+  icon: tags
+  summary: >
     Edit tag metadata (excerpt and wiki) and review suggested edits to tags
   post_score_threshold: null
   edit_score_threshold: 0.975   # min. +38/-0
   flag_score_threshold: null
 
 - name: Vote on Holds
   internal_id: flag_close
-  icon: fas fa-lock
-  description: >
+  icon: lock
+  summary: >
     Vote to put questions on hold, vote to reopen on holded questions and review hold votes

```

Comment:
Vote to put questions on hold, see pending hold votes, and vote to keep open or reopen.

(I know we don't have our new hold UI yet with the running tallies, but let's plan for that.  Also, we'd need to change "on holded" anyway.)

Reply:
You don't like "on holded"? :P